### ISSUE TITLE
Is DaisyRec going to have session-based recommenders?

### ISSUE THREAD

**Issue by ita9naiwa** [AmazingDD/daisyRec#2]
First of all, I really appreciate your work and this library. It's what exactly I've looking for..!

I see your paper ends with "diving into more diverse recommendation tasks (e.g., session-aware)".

I just wondered that this library is going to include session based recommenders such as GRURec(https://arxiv.org/abs/1511.06939)...

**Comment by yudi-mars** [comment]
Thx for your letter. We firstly decided to implement session-aware algorithms, but soon we found that these methods are not adapt to this daisyRec framework. Therefore, we except these algorithms and only involve ranking-related algorithms. In the future, we decided to create another library which are only relative to session-based recommendation algorithms, but it is still under discussion.&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "AmazingDD/daisyRec"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月6日(星期二) 晚上11:10
收件人:&nbsp;"AmazingDD/daisyRec"<daisyRec@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[AmazingDD/daisyRec] Is DaisyRec going to have session-based recommenders? (#2)





 
First of all, I really appreciate your work and this library. It's what exactly I've looking for..!
 
I see your paper ends with "diving into more diverse recommendation tasks (e.g., session-aware)".
 
I just wondered that this library is going to include session based recommenders such as GRURec(https://arxiv.org/abs/1511.06939)...
